lib/Runtime/Base/HiResTimer.h

```diff
@@ -22,7 +22,14 @@ namespace Js
 
         double GetAdjustFactor();
     public:
-        HiResTimer(): fInit(false), dBaseTime(0), baseMsCount(0),  fHiResAvailable(true), dLastTime(0), dAdjustFactor(1), fReset(true) {}
+        HiResTimer(): fInit(false), dBaseTime(0), baseMsCount(0),  
+#ifdef _WIN32
+        // xplat-todo: do we really need this for now?
+        fHiResAvailable(true), 
+#else
+        fHiResAvailable(false), 
```

Comment:
I dont understand- it looks like the PAL provides QueryPerformanceCounter, yes? Then why are we ifdef-ing here, the code in HiResTimer::Now should work fine, no?